tests/toranj/wpan.py

```diff
@@ -434,7 +434,12 @@ def status(self):
         return self.wpanctl('status')
 
     def leave(self):
-        return self.wpanctl('leave')
+        res = self.wpanctl('leave')
+        for _ in range(5):
+            if self.get(WPAN_STATE) != STATE_OFFLINE:
+                print('***WARN***: Not in offline state')
+                time.sleep(1)
+        return res
```

Comment:
@gjc13 I think this is related to discussions from https://github.com/openthread/wpantund/pull/444#discussion_r349445074


I am not sure if such a fix to wait for it in test script would be the right approach. I expect the `wpantund` itself would do the same and if it does not then I would say we'd need to address it in `wpantund` itself.

Checking the code, the wpantund leave task does seem to have code that waits for NCP to finish initialization:
https://github.com/openthread/wpantund/blob/9d31a884ef2618776df784e1eec1cc8c25f31943/src/ncp-spinel/SpinelNCPTaskLeave.cpp#L111-L132

Also in the wpantund form task, we seem to wait for some time for NCP to be ready before proceeding:
https://github.com/openthread/wpantund/blob/9d31a884ef2618776df784e1eec1cc8c25f31943/src/ncp-spinel/SpinelNCPTaskForm.cpp#L130-L135